app/controllers/v3/service_credential_bindings_controller.rb

```diff
@@ -119,7 +119,7 @@ def parameters
 
   def create_key_binding(message, service_instance)
     action = V3::ServiceCredentialBindingKeyCreate.new(user_audit_info, message.audit_hash)
-    binding = action.precursor(service_instance, message.name)
+    binding = action.precursor(service_instance, message.name, message: message)
```

Comment:
Perhaps it's not necessary to send `message` and `message.name`

Reply:
Fixed this in latest commit